packages/contracts-bedrock/test/L2/SuperchainERC20.t.sol

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+// Testing utilities
+import { Test } from "forge-std/Test.sol";
+
+// Libraries
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { IERC20 } from "@openzeppelin/contracts-v5/token/ERC20/IERC20.sol";
+import { IL2ToL2CrossDomainMessenger } from "src/L2/interfaces/IL2ToL2CrossDomainMessenger.sol";
+
+// Target contract
+import {
+    CallerNotL2ToL2CrossDomainMessenger,
+    InvalidCrossDomainSender,
+    SuperchainERC20,
+    ISuperchainERC20Extensions,
+    ZeroAddress
+} from "src/L2/SuperchainERC20.sol";
+
+/// @notice Mock contract for the SuperchainERC20 contract so tests can mint tokens.
+contract SuperchainERC20Mock is SuperchainERC20 {
+    string private _name;
+    string private _symbol;
+    uint8 private _decimals;
+
+    constructor(string memory __name, string memory __symbol, uint8 __decimals) {
+        _name = __name;
+        _symbol = __symbol;
+        _decimals = __decimals;
+    }
+
+    function mint(address _account, uint256 _amount) public {
+        _mint(_account, _amount);
+    }
+
+    function name() public view virtual override returns (string memory) {
+        return _name;
+    }
+
+    function symbol() public view virtual override returns (string memory) {
+        return _symbol;
+    }
+
+    function decimals() public view virtual override returns (uint8) {
+        return _decimals;
+    }
+}
+/// @title SuperchainERC20Test
+/// @notice Contract for testing the SuperchainERC20 contract.
+
```

Comment:
nit: will solc still parse this even though there is a newline?

Reply:
This line is something `forge fmt` adds. I tried removing and pushing, and the linter failed.